nsxt/resource_nsxt_transit_gateway_test.go

```diff
@@ -0,0 +1,179 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestTransitGatewayCreateAttributes = map[string]string{
+	"display_name":    getAccTestResourceName(),
+	"description":     "terraform created",
+	"transit_subnets": "192.168.5.0/24",
+}
+
+var accTestTransitGatewayUpdateAttributes = map[string]string{
+	"display_name":    getAccTestResourceName(),
+	"description":     "terraform updated",
+	"transit_subnets": "192.168.7.0/24",
+}
+
+func TestAccResourceNsxtTransitGateway_basic(t *testing.T) {
```

Comment:
Note that vpc20 CI job doesn't run those tests now, as only tests that match `^TestAcc.*V[Pp][Cc].*$` regexp are executed

Reply:
Thanks, good to know. Can we add this specific one to the list? I'm not sure about the naming btw